### PR TITLE
https://ioetec.atlassian.net/browse/TT-656

### PR DESCRIPTION
**Pull request by rodolfoIOET**
Configure time-tracker-ui through CD pipeline for both: stage and prod

### CODE REVIEW

File: .github/workflows/time-tracker-ui-ci.yml [hunk 1]
```diff
@@ -77,13 +89,17 @@ jobs:
...
-      working-directory: ./${{ env.WORKING_DIR }}
-      env:
-        TF_WORKSPACE: prod
+    ##################### THIS SECTION IS COMMENTED BECAUSE PRODUCTION IS SHUTDOWN IN AZURE ###########
+    ##################### PLEASE UNCOMMENT BELOW ONCE AZURE PRODUCTION IS READY #######################
+
+
+    # - name: Terraform Plan Prod
+    #   id: plan-stage
+    #   run: terraform plan -var-file=${{ env.TF_WORKSPACE }}.tfvars -var image_tag=latest -no-color
```
Comment (faustocv) [inline .github/workflows/time-tracker-ui-ci.yml:98]: Could you just comment out the run line and add a echo message like: 
```
 run: echo "Disabled for now up to restructure infra tiers"
```

File: .github/workflows/time-tracker-ui-ci.yml [hunk 2]
```diff
@@ -108,32 +124,36 @@ jobs:
...
-          \`\`\`
-          </details>
-          *Pusher: @${{ github.actor }}, Action: \`${{ github.event_name }}\`*`;
-          github.issues.createComment({
-            issue_number: context.issue.number,
-            owner: context.repo.owner,
-            repo: context.repo.repo,
-            body: output
-          })
+
```
Comment (faustocv) [inline .github/workflows/time-tracker-ui-ci.yml:127]: Uncomment all of this

File: .github/workflows/time-tracker-ui-ci.yml [hunk 3]
```diff
@@ -108,32 +124,36 @@ jobs:
...
+    #         body: output
+    #       })
     - name: Terraform Plan Stage Status
       if: steps.plan-stage.outcome == 'failure'
       run: exit 1
 
-    - name: Terraform Plan Prod Status
-      if: steps.plan-prod.outcome == 'failure'
-      run: exit 1
+    # - name: Terraform Plan Prod Status
```
Comment (faustocv) [inline .github/workflows/time-tracker-ui-ci.yml:157]: Uncomment all of this

**Review by faustocv — APPROVED** [review]

**Review by JaviMiot — APPROVED** [review]
great job

**Comment by rodolfoIOET** [comment]
solved by https://ioetec.atlassian.net/browse/TTA-87

**Pull request closed without merging**